diff --git a/cmake/DakotaFindPython.cmake b/cmake/DakotaFindPython.cmake
index de128fc8..3410fa7f 100644
--- a/cmake/DakotaFindPython.cmake
+++ b/cmake/DakotaFindPython.cmake
@@ -34,12 +34,6 @@ macro(dakota_find_python)
     # TODO: fine-grained error messages
     # if(DAKOTA_PYTHON_DIRECT_INTERFACE and NOT Python_Development_FOUND)
 
-    if(DAKOTA_PYTHON_SURROGATES)
-      # This add_subdirectory must be done at top-level so pybind11's
-      # CMake functions are pulled in for src/ and below
-      add_subdirectory(packages/external/pybind11)
-    endif()
-
   else()
 
     # Disable some components that definitely won't work
