diff --git a/src/PluginInterface.cpp b/src/PluginInterface.cpp
index 60f4008..d268dfc 100644
--- a/src/PluginInterface.cpp
+++ b/src/PluginInterface.cpp
@@ -111,7 +111,7 @@ void PluginInterface::load_plugin()
 {
   if (pluginInterface) return;
   try {
-    pluginInterface = boost::dll::import<DakotaPlugins::DakotaInterfaceAPI>
+    pluginInterface = boost::dll::import_symbol<DakotaPlugins::DakotaInterfaceAPI>
       (pluginPath,
 	 "dakota_interface_plugin"  // name of the symbol to import
 	 // TODO: append .dll, .so, .dylib via
