diff --git a/src/unit_test/CMakeLists.txt b/src/unit_test/CMakeLists.txt
index 166158f35..364c68d3f 100644
--- a/src/unit_test/CMakeLists.txt
+++ b/src/unit_test/CMakeLists.txt
@@ -84,7 +84,8 @@ dakota_add_unit_test(NAME lhs_constants
   SOURCES lhs_constants.cpp
   LINK_DAKOTA_LIBS
   )
-target_link_libraries(lhs_constants Boost::boost)
+
+
 
 # Unit test: h5py_hdf5
 if(DAKOTA_H5PY_FOUND)
@@ -116,6 +117,15 @@ dakota_add_unit_test(NAME response_io
   )
 target_link_libraries(response_io Boost::boost)
 
+target_link_libraries(lhs_constants Boost::boost)
+target_link_libraries(min_unit_test "rt")
+target_link_libraries(auto_unit_test "rt")
+target_link_libraries(bootstrap_util_test "rt")
+target_link_libraries(test_field_covariance_utils "rt")
+target_link_libraries(lhs_constants "rt")
+target_link_libraries(response_io "rt")
+target_link_libraries(redirect_regexs "rt")
+
 # Unit test: experiment data and readers
 # Demonstration of Teuchos test framework to driver several tests related to
 # ExperimentData and associated file readers
